### PR TITLE
Bump Beam version to 2.60.0

### PR DESCRIPTION
**Pull request by Abacn**

### CODE REVIEW

**Comment by Abacn** [comment]
failed assert is due to the good file now is named after

```
Will copy temporary file FileResult{tempFilename=gs://cloud-teleport-testing-it-gitactions/YAMLTemplateIT/20241018-6698b103-f0fb-475f-b4b2-53ba38f49668/output/.temp-beam/0a421716c734af1d-e70e-4718-ae63-54e8c065ab08, shard=0, window=GlobalWindow, paneInfo=PaneInfo.NO_FIRING} to final location gs://cloud-teleport-testing-it-gitactions/YAMLTemplateIT/20241018-6698b103-f0fb-475f-b4b2-53ba38f49668/output/goodGlobalWindow-pane-0-last-00000-of-00001
```

File: python/src/test/java/com/google/cloud/teleport/templates/python/YAMLTemplateIT.java [hunk 1]
```diff
@@ -77,7 +77,7 @@ private void testSimpleComposite(
     runYamlTemplateTest(paramsAdder);
 
     // Assert
-    List<Artifact> goodArtifacts = gcsClient.listArtifacts("output/good-", Pattern.compile(".*"));
```
Comment (Abacn) [inline python/src/test/java/com/google/cloud/teleport/templates/python/YAMLTemplateIT.java:80]: @Polber this is the same issue during release validation - please check

**Comment by Abacn** [comment]
R: @Polber @damccorm

**Review by damccorm — APPROVED** [review]
Thanks!

**Pull request merged**